EIPS/eip-1319.md

```diff
@@ -166,6 +166,63 @@ release versions while allowing them to use any versioning schema they choose.
 
 Registries may offer more complex `read` APIs that manage requests for packages within a semver range or at `latest` etc. This EIP is agnostic about how tooling or registries might implement these. It recommends that registries implement [EIP 165](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-165.md) and avail themselves of resources to publish more complex interfaces such as [EIP 926](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-926.md).
 
+## EthPM URIs
```

Comment:
I'm not fully sure the best way is to include both the URI format and everything else in a single EIP. Would suggest for consideration to make this a separate EIP depending on this one.